snippets/reduceWhich.md

```diff
@@ -0,0 +1,25 @@
+### reduceWhich
+
+By setting some rules, get Minimum or Maximum value in array .
+
+BTW, the Minimum / Maximum value could be `Object`, namely, the upper array could be a `Object` array(filled with `Object`).
```

Comment:
The explanation of the snippet should briefly mention native methods and techniques used to achieve this functionality.

Reply:
Fixed